docs/reference/generated/accordion-header.json

```diff
@@ -3,11 +3,11 @@
   "description": "A heading that labels the corresponding panel.\nRenders an `<h3>` element.",
   "props": {
     "className": {
-      "type": "string | (state) => string",
+      "type": "string | ((state: State) => string)",
       "description": "CSS class applied to the element, or a function that\nreturns a class based on the component’s state."
     },
     "render": {
-      "type": "React.ReactElement | (props, state) => React.ReactElement",
+      "type": "ReactElement | ((props: GenericHTMLProps, state: State) => ReactElement)",
```

Comment:
Was it intentional that 'React.' is removed as a prefix?

Reply:
Yes, I decided to display just the type name, as the fully qualified name sometimes looks incorrect - it contains the path to the file it's declared in. I played it safe by using just the type name.